src/TrustedShops.php

```diff
@@ -82,6 +82,28 @@ public static function woocommerce_thankyou_order_received_text($text, $order) {
   <span id="tsCheckoutOrderAmount">{$order->get_total()}</span>
   <span id="tsCheckoutOrderCurrency">{$order->get_currency()}</span>
   <span id="tsCheckoutOrderPaymentType">{$order->get_payment_method()}</span>
+EOD;
+    foreach ($order->get_items() as $item) {
+      $product = $item->get_product();
+      $product_id = $product->get_id();
+      $product_url = get_permalink($product_id);
+      $product_image = has_post_thumbnail($product_id) ? wp_get_attachment_url($product->get_data()['image_id']) : '';
+      $product_sku = $product->get_sku();
+      $product_gtin = get_post_meta($product_id, '_custom_gtin', TRUE);
```

Comment:
The GTIN field is custom and does not exist on all sites using woocommerce-reputations.  Do we need to handle this?

Reply:
We're using GTIN on almost every our e-commerce website. The GTIN feature will be also moved to the new ECOMS shop-standards plugins. 

Aside from that, yes — Stephan explicitly asked to integrate GTIN into TS reviews.